tools/templates/ci.yml

```diff
@@ -50,12 +50,34 @@ jobs:
       command: build
       projects: '$(file)'
       arguments: '--configuration Release'
+  - task: DotNetCoreCLI@2
+    inputs:
```

Comment:
Please add a descriptive `displayName` to the task. It shows up as `DotNetCoreCLI` now.